python/services/dataservice/dmod/dataservice/__main__.py

```diff
@@ -0,0 +1,96 @@
+import argparse
+from . import name as package_name
+from .service import ServiceManager
+from pathlib import Path
+from socket import gethostname
+
+
+def _handle_args():
+    parser = argparse.ArgumentParser(formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+    parser.add_argument('--host',
+                        help='Set the appropriate listening host name or address value (NOTE: must match SSL cert)',
```

Comment:
Anything more than a couple args makes we wonder if we shouldn't move to a config file (yaml?)...

Reply:
Possibly, but that will also require either:
1. a mechanism for generating such files dynamically when starting stacks, from the configured project-level environment variables, or 
2. that users be responsible for edit multiple config files, likely with there being some things edited in more than one place that then manually need to be kept in sync

The project-level configuration mechanisms are in need of a significant overhaul, at which point something like _1._ would be easier.  For now though, I don't immediately see another way of keeping a single source of config settings.